packages/base/src/util/ColorConversion.ts

```diff
@@ -196,6 +196,19 @@ const getRGBColor = (color: string): ColorRGB => {
 	return HEXToRGB(color);
 };
 
+const getAlpha = (color: string): number => {
+	let alpha = 1;
+
+	if (color.startsWith("rgba")) {
+		const parts = color.split(",");
```

Comment:
Also for a solution, we could use reggex if we only want rgba:

const rgbaRegex = /^rgba\(\s*\d+\s*,\s*\d+\s*,\s*\d+\s*,\s*([\d.]+)\s*\)$/;
const match = rgbaString.match(rgbaRegex);

    if (match) {
        return parseFloat(match[1]);
    }

But that's if we don't have hsla and rgba

Reply:
I am willing to keep the non-regex method as it is more readable.